compiler/codegen/CCData.cpp

```diff
@@ -41,7 +41,6 @@ size_t CCData::dataAlignmentFromBytesAlignment(size_t alignmentBytes)
    return (alignmentBytes + OMR_ALIGNOF(data_t) - 1) / OMR_ALIGNOF(data_t);
    }
 
-inline
```

Comment:
@ymanton @0xdaryl I had to remove the `inline` keyword to resolve missing symbol errors:
```
JVMJ9VM011W Unable to load j9jit29: /Users/heidinga/projects/openj9-openjdk-jdk8/build/linux-x86_64-normal-server-release/images/j2sdk-image/jre/lib/amd64/compressedrefs/libj9jit29.so: undefined symbol: _ZN2TR6CCData22byteIndexFromDataIndexEm
```